packages/accounts/plugindefs/multi-owner/config.ts

```diff
@@ -7,9 +18,17 @@ export const MultiOwnerPluginGenConfig: PluginGenConfig = {
   name: "MultiOwnerPlugin",
   abi: MultiOwnerPluginAbi,
   addresses: {
+    [mainnet.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
+    [optimism.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
+    [arbitrum.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
+    [polygon.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
+    [base.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
+
     [sepolia.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
+    [optimismSepolia.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
+    [arbitrumSepolia.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
+    [polygonMumbai.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
     [baseSepolia.id]: "0x000000E8F14A838A00505d861c6EF15cdfB05455",
```

Comment:
Should this be the address for `MultiOwnerPlugin`? If so, this address is incorrect, it should be `0xcE0000007B008F50d762D155002600004cD6c647` for all networks.